ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -198,7 +198,7 @@ objects:
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
           -e openshift_release="${PREVIOUS_ANSIBLE_VERSION}" \
-          -e openshift_version="${PREVIOUS_ANSIBLE_VERSION}.0" \
```

Comment:
You're not only skipping something but also not providing this -e var any more -- is that intended?